File: src/fr/cph/stock/android/domain/ResponseDTO.java
package fr.cph.stock.android.domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
public class ResponseDTO {
	private Portfolio portfolio;

	private String error;

	public Portfolio getPortfolio() {
		return portfolio;
	}

	public void setPortfolio(Portfolio portfolio) {
		this.portfolio = portfolio;
	}

	public String getError() {
		return error;
	}

	public void setError(String error) {
		this.error = error;
	}
}

File: src/fr/cph/stock/android/domain/ResponseDTO.kt
package fr.cph.stock.android.domain

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude
import kotlin.properties.Delegates

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
class ResponseDTO {
    var portfolio: Portfolio by Delegates.notNull()

    var error: String? = null
}
